### PR TITLE
Sticky Notes integrated to skeleton, including unit tests and local storage

### LINKED ISSUE #2: Integrate Sticky Notes into Skeleton

### LINKED ISSUE #15: Sticky Notes can be seen outside of middle div box
You can drag sticky notes outside of the middle area for sticky notes, ideally you cannot see/drag them outside.

### PR DESCRIPTION
**Pull request by CalvQ**

### CODE REVIEW

**Review by yuz876 — CHANGES_REQUESTED** [review]
I tried it locally. But sth wired. I think it is possible your css affect it. Could you check again?

image

File: source/css/stickynotes.css [hunk 1]
```diff
@@ -0,0 +1,83 @@
+@import url("https://fonts.googleapis.com/css2?family=Nunito+Sans:wght@300&display=swap");
+
+body {
```
[the remark this answers is not in the document]
Comment (CalvQ) [reply source/css/stickynotes.css]: Good catch I should not have done that, fixed now

**Comment by yuz876** [comment]
I have tried again and deleted the line 3 to 25 in your css (body, *, button) and the sticky notes part works. Great job Calvin! I think you can just fix it by deleting line 3 to 25.


Also , looks like you  delete the calendar section by mistake? It doesn't have calendar section but has two mood tracker summary section. Would you plz fix it also?

**Comment by yuz876** [comment]
It looks great! Well done Calvin! I will merge it.

Also it has a small bug but I will merge this one firstly. the small bug can be fixed in your future PR. After. I click refresh, the text of sticky notes became less. The second screenshot below is after refresh.

Screen Shot 2021-06-03 at 21 10 46



Screen Shot 2021-06-03 at 21 11 03

**Pull request merged**